[testing] Introduce unit tests for SequencerInbox.sol (#38)
SpecularL2/specular: contracts/README.md

## Run Tests

```sh
forge test
```

For increased levels of test output verbosity, increase the number of `v`s. For example:
```sh
forge test -v
```
or
```sh
forge test -vv
```
or
```sh
forge test -vvv
```
or
```sh

> forge test -vvvv

Author: Sure. Updated.

```

## Run Lints

This is not really required, as formatting happens everytime you commit to the repository.

```sh
forge fmt .

> ```

Reviewer: nit: i'd just get rid of this section since it's automated, and we can introduce a separate readme for contributions

Author: Sure. Removed. We can later discuss what needs to be included in the `Contributions` section.

## Fuzz Runs

> To increase the number of runs for the fuzz tests written for testing `Rollup.t.sol`, `SequencerInbox.t.sol`, etc. head over to `foundry.toml` and under the `[fuzz]` section put your desired number of runs infront of `runs`.

Reviewer: nit: consider
```suggestion
The number of fuzz test runs can be configured in `foundry.toml`, under the `[fuzz]` section with parameter `runs`.
```
nit: move this under `Run Tests`; avoid naming specific `t.sol` files here since the coverage will grow

Author: Sure.
